### ISSUE TITLE
ifGesture orGate seems to imply noConsume

### ISSUE THREAD

**Issue by mhantsch** [UltimateHackingKeyboard/firmware#800]
macro **`testGesture`**:

```
ifGesture timeoutIn 1500 k final tapKey x
ifGesture timeoutIn 1500 orGate l final tapKey y
```

Bind to `fn`+`x`.

Press `fn`+`x` followed by `k` => taps **x**
Press `fn`+`x` followed by `l` => taps **ly**

Seems like orGate also triggers noConsume, but I don't think it should. At least that's not how I understand the docs.

**Comment by kareltucek** [comment]
I agree with your interpretation. Will look into it, but it may need to wait.

Thanks for report!

**Comment by mondalaci** [comment]
Fixed by #1205.
